_posts/2020-07-30-axon-kompleksowe-testowanie-aplikacji.md

```diff
@@ -0,0 +1,386 @@
+---
+layout:    post
+title:     "Axon - Kompleksowe testowanie aplikacji"
```

Comment:
Masz bardzo dobry tytuł artykułu, ale żeby łatwiej trafić na niego z poziomu wyszukiwarki, spróbuj dodać do któregoś akapitu hasło "Axon". To zwiększy szanse, że artykuł przesunie się wyżej w hierarchii.

Reply:
Jeden z akapitów zmieniłem na `Testy integracyjne - możliwe z Axonem?`. Good enough?

Reply:
Bardzo dobrze.